keras/backend.py

```diff
@@ -6117,10 +6089,10 @@ def conv2d_transpose(
         raise ValueError("Unknown data_format: " + str(data_format))
 
     # `atrous_conv2d_transpose` only supports NHWC format, even on GPU.
-    if data_format == "channels_first" and dilation_rate != (1, 1):
-        force_transpose = True
-    else:
-        force_transpose = False
+    force_transpose = data_format == "channels_first" and dilation_rate != (
```

Comment:
The format here is quite weird, how about wrap the both cause in a () and break at the `and`

Reply:
oh yeah! i saw that after running `sh shell/format.sh` and was surprised it made it like that

Reply:
updated